observability-test/database.ts

```diff
@@ -375,6 +375,115 @@ describe('Database', () => {
     });
   });
 
+  describe('batchCreateSessions', () => {
+    it('without error', done => {
```

Comment:
```suggestion
    it('should execute batchCreateSessions without error', done => {
```

Reply:
This feels pedantic/verbose and repeating also out of context because I am really testing what happens without an error returned and also with an error returned.

Reply:
We aren't testing Cloud Spanner's code but rather the instrumentation added and what happens with and without errors.